app/nexus/internal/initialization/recovery/shamir.go

```diff
@@ -80,6 +81,10 @@ func computeShares() (group.Scalar, []shamir.Share) {
 	defer state.UnlockRootKey()
 	rk := state.RootKeyNoLock()
 
+	if rk == nil || mem.Zeroed32(rk) {
+		log.FatalLn(fName, "message", "root key is nil or zeroed")
+	}
```

Comment:
getting a bad root key here is either a configuration, or a security error. It's best to crash loud and clear since this is unexpected.